src/main/java/PageFactory.java

```diff
@@ -0,0 +1,11 @@
+import Pages.*;
+import java.io.IOException;
+import static com.codeborne.selenide.Selenide.open;
+
+public class PageFactory {
+
+    public static LoginPage openLoginPage() throws IOException {
```

Comment:
please, try not to repeat the same mistakes. what's the point to just throw exceptions above context?